tests/client_test.go

```diff
@@ -363,12 +369,18 @@ func TestUpstreamServer(t *testing.T) {
 	}
 
 	maxFails := 64
+	weight := 10
 	server := client.UpstreamServer{
 		Server:      "127.0.0.1:2000",
 		MaxConns:    321,
 		MaxFails:    &maxFails,
 		FailTimeout: "21s",
 		SlowStart:   "12s",
+		Weight:      &weight,
+		Route:       "test",
+		Drain:       false,
+		Backup:      true,
+		Down:        true,
 	}
```

Comment:
Is it possible to add test coverage for the new `Service` parameter?

Reply:
Not possible, it depends on `resolve` in `http` block.